tests/integration/test_azure_storage.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+from tests.integration.conftest import _create_ensemble, _create_experiment
```

Comment:
Importing from `conftest` is bad manners

Reply:
I agree